keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/Dates.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.clouddriver
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+class Dates {
+  companion object {
+    private val formats: List<DateTimeFormatter> = listOf(
```

Comment:
Is this because our images are tagged inconsistently? Because… 😨

Reply:
This is pulled from swabbie because I spent ~10 minutes trying to parse the date w/o it and then got very annoyed and took what we're already doing.